js/main.js

```diff
@@ -112,6 +112,10 @@ function entryTemplate(entry) {
   $columnNote.setAttribute('class', 'column-full');
   $note.textContent = entry.note;
 
+  for (var x = 0; x < $li.length; x++) {
+    $li[x].setAttribute('data-entry-id', data.entries[x].entryId);
+  }
```

Comment:
You shouldn't need a loop, you just need to add it to the entry currently being created. You'll add a `data-entry-id` attribute to your `$entryContainer` element in this function.